apps/payments/app/components/AnalyticsTracker.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import { usePathname } from "next/navigation";
+import { useEffect, useRef } from "react";
+import { getBuildingBlockSDK } from "@ogcio/building-blocks-sdk";
+
+const buildingBlocksFrontend = getBuildingBlockSDK({
+  services: {
+    analytics: {
+      baseUrl: process.env.NEXT_PUBLIC_ANALYTICS_URL!,
+      matomoToken: process.env.NEXT_PUBLIC_ANALYTICS_MATOMO_TOKEN,
+      trackingWebsiteId: process.env.NEXT_PUBLIC_ANALYTICS_WEBSITE_ID,
+      organizationId: process.env.NEXT_PUBLIC_AUTH_ORGANIZATION_ID!,
+    },
```

Comment:
Avoid exposing Matomo Token on client side

```suggestion
    analytics: {
      baseUrl: process.env.NEXT_PUBLIC_ANALYTICS_URL!,
      trackingWebsiteId: process.env.NEXT_PUBLIC_ANALYTICS_WEBSITE_ID,
      organizationId: process.env.NEXT_PUBLIC_AUTH_ORGANIZATION_ID!,
    },
```